### ISSUE TITLE
Use a symbolic link to node_modules instead of copying?

### ISSUE THREAD

**Issue by cgewecke** [sc-forks/solidity-coverage#105]
In #103 @eenae solved their problem accessing npm installed contracts by generating a symbolic link from the target's node_modules to `coverageEnv`. Tried this once from within `app.js` using the shell module and couldn't get it to work. . .but possibly due to incompetence.

If this can be done and has no weird side effects, it seems like a nice enhancement.

**Comment by area** [comment]
This definitely should be possible...  I would expect something like: 

```
shell.ln('-s', `${this.workingDir}/node_modules`, `${this.coverageDir}/node_modules`)
```
to work?

**Comment by cgewecke** [comment]
@area Cool.

**Comment by cgewecke** [comment]
Resolved with #157.
